napalm/ios/ios.py

```diff
@@ -377,6 +382,33 @@ def compare_config(self):
 
         return diff.strip()
 
+    def _file_operation(f):
```

Comment:
Let's rename this to something like `_file_prompt_quiet`. `_file_operation` is too generic.

Reply:
Ack